doc/overview.rst

```diff
@@ -1,33 +1,32 @@
 Overview
 =========
 
-SaltProc couples directly with the Monte Carlo neutron transport code Serpent 2
-and enables sophisticated, multi-component online fuel reprocessing system
+SaltProc couples directly with a variety of Monte Carlo neutron transport codes with depletion capabilities and enables sophisticated, multi-component online fuel reprocessing system
 modeling. SaltProc is the first open-source tool for liquid-fueled MSR
 depletion simulation with the following capabilities:
 
 - neutron poison removal with user-defined efficiency,
 - make-up mass loss by fresh fuel injection,
-- reactivity control by adjusting fuel feed rate or geometry change,
+- reactivity control by adjusting fuel feed rate or geometry,
 - can model any reactor design,
 - potentially, can couple with any depletion tool (i.e., MCNP, SCALE, OpenMC).
 
 
 How SaltProc works
 -------------------
 
-SaltProc is a driver for Serpent to simulate online fuel salt reprocessing for
+SaltProc is a driver for Monte Carlo depletion codes (henceforth referred to as *depletion codes*) to simulate online fuel salt reprocessing for
 Molten Salt Reactors. It performs following major functions:
 
-  - runs SERPENT
-  - parses and stores Serpent output data in HDF5
+  - runs the depletion code
```

Comment:
When I think of running a depletion code, I wouldn't consider the Monte Carlo transport as part of the depletion step. Maybe this could be clarified here by "runs the Monte Carlo transport and depletion" or something?

Reply:
That's a good point. I'll add that.